pkg/clusters/addons/kong/kong.go

```diff
@@ -296,3 +336,29 @@ func urlForService(ctx context.Context, cluster clusters.Cluster, nsn types.Name
 
 	return nil, fmt.Errorf("service %s has not yet been provisoned", service.Name)
 }
+
+func deployKongEnterpriseLicenseSecret(ctx context.Context, cluster clusters.Cluster, namespace, name string) error {
+	license := os.Getenv(EnterpriseLicense)
+	if license == "" {
+		return fmt.Errorf("failed retrieving license key from environment")
+	}
+
+	fmt.Printf("removing me %s", license)
+	newSecret := &corev1.Secret{
+		Type: corev1.SecretTypeOpaque,
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Data: map[string][]byte{
+			"license": []byte(license),
+		},
+	}
+
+	_, err := cluster.Client().CoreV1().Secrets(namespace).Create(ctx, newSecret, metav1.CreateOptions{})
+	if err != nil {
+		return fmt.Errorf("failed creating kong-enterprise-license secret, err %v", err)
```

Comment:
`error` vars should be wrapped with `%w` formatting:

```suggestion
		return fmt.Errorf("failed creating kong-enterprise-license secret, err %w", err)
```